src/content/docs/learn-more/configuration-variables.md

```diff
@@ -41,21 +41,23 @@ By default, configuration variables get their values from environment variables.
 :::tab{value=npm}
 
 ```bash
-SEPOLIA_RPC_URL='https://eth-sepolia.g.alchemy.com/v2/ABC123...' npx hardhat test
+SEPOLIA_RPC_URL='<https://eth-sepolia.g.alchemy.com/v2/ABC123>...' npx hardhat run ./my-script.ts --network sepolia
 ```
 
 :::
 
 :::tab{value=pnpm}
 
 ```bash
-SEPOLIA_RPC_URL='https://eth-sepolia.g.alchemy.com/v2/ABC123...' pnpm hardhat test
+SEPOLIA_RPC_URL='<https://eth-sepolia.g.alchemy.com/v2/ABC123>...' pnpm hardhat run ./my-script.ts --network sepolia
 ```
 
 :::
 
 ::::
 
+> The command is just an example, the `SEPOLIA_PRIVATE_KEY` must be also defined and `my-script.ts` implemented.
```

Comment:
I would drop this. The fact that `my-script.ts` must be implemented doesn't need clarification, and it's perfectly valid to have a read-only script that doesn't need a private key.